xml/mysql_banners.xml

```diff
@@ -0,0 +1,736 @@
+<?xml version="1.0"?>
+<fingerprints matches="mysql.banners">
+
+   <fingerprint pattern="^(\d\.\d{1,2}\.\h{1,3}(?:[.-]\d{1,2})?(?:[.-]\d{1})?)(?:-m\d{1,2})?(?:-rc)?(?:-alpha)?(?:-beta)?(?:-gamma)?(?:-?[Mm]ax)?(?:-rs)?(?:-modified)?(?:-debug)?(?:-log)?$">
```

Comment:
Not sure about MySQL proper, but MariaDB uses 10.x.x versions as they diverge from Oracle's implementation.